resources/dashboards/Traffic-Endpoints-Dashboard.json

```diff
@@ -3,9 +3,9 @@
    "timeRestore": false, 
    "description": "", 
    "title": "Traffic Endpoints Dashboard", 
-   "panelsJSON": "[{\"id\":\"Support-Table\",\"type\":\"search\",\"size_x\":6,\"size_y\":4,\"col\":1,\"row\":1,\"columns\":[\"Application\",\"SrcIP\",\"DestIP\",\"Duration\"],\"sort\":[\"TimeUpdated\",\"asc\"]},{\"id\":\"Top-SrcIP-By-Metadata-Impact-(pie)\",\"type\":\"visualization\",\"size_x\":3,\"size_y\":2,\"col\":7,\"row\":1},{\"id\":\"Top-DestIP-By-Metadata-Impact-(pie)\",\"type\":\"visualization\",\"size_x\":3,\"size_y\":2,\"col\":7,\"row\":3},{\"id\":\"Top-DestIP-By-Session-(pie)\",\"type\":\"visualization\",\"size_x\":3,\"size_y\":2,\"col\":10,\"row\":3},{\"id\":\"Top-SrcIP-By-Session-(pie)\",\"type\":\"visualization\",\"size_x\":3,\"size_y\":2,\"col\":10,\"row\":1}]", 
+   "panelsJSON": "[{\"id\":\"Support-Table\",\"type\":\"search\",\"size_x\":12,\"size_y\":6,\"col\":1,\"row\":3,\"columns\":[\"Application\",\"SrcIP\",\"DestIP\",\"Duration\"],\"sort\":[\"TimeUpdated\",\"asc\"]},{\"id\":\"Top-SrcIP-By-Metadata-Impact-(pie)\",\"type\":\"visualization\",\"size_x\":3,\"size_y\":2,\"col\":1,\"row\":1},{\"id\":\"Top-DestIP-By-Metadata-Impact-(pie)\",\"type\":\"visualization\",\"size_x\":3,\"size_y\":2,\"col\":4,\"row\":1},{\"id\":\"Top-DestIP-By-Session-(pie)\",\"type\":\"visualization\",\"size_x\":3,\"size_y\":2,\"col\":10,\"row\":1},{\"id\":\"Top-SrcIP-By-Session-(pie)\",\"type\":\"visualization\",\"size_x\":3,\"size_y\":2,\"col\":7,\"row\":1}]", 
    "version": 2, 
```

Comment:
For both dashboards, you need to increment the version to 3 for the insert script to upgrade them properly. Otherwise, it will see the same version number and ignore them.